### ISSUE TITLE
Release 3.1.1 does not allow you to revert to ealier editions of any content in Plone 5.1

### ISSUE THREAD

**Issue by lheagney** [plone/Products.CMFEditions#50]
I've updated three different sites from plone 5.0.8 to plone 5.1 and all of them had the same problem. I could view the history for all content, I could compare, I could view the previews, but when I clicked on any of the revert buttons I received an "Insufficient Privileges" error.

When I disabled the exception filtering in the error_log I can see the following error produced each time I attempted to revert: 
```

Traceback (innermost last):
Module ZPublisher.Publish, line 138, in publish
Module ZPublisher.mapply, line 77, in mapply
Module ZPublisher.Publish, line 48, in call_object
Module Products.CMFCore.FSPythonScript, line 127, in __call__
Module Shared.DC.Scripts.Bindings, line 322, in __call__
Module Shared.DC.Scripts.Bindings, line 359, in _bindAndExec
Module Products.PythonScripts.PythonScript, line 344, in _exec
Module script, line 4, in revertversion
<FSPythonScript at /ipes-cdt/revertversion used for /ipes-cdt/front-page>
- Line 4
Module AccessControl.ZopeGuards, line 305, in guarded_import
Unauthorized: import of 'six' is unauthorized
```

Pinning Products.CMFEditions to version 3.1 in buildout fixes this issue, so it looks like this is something introduced in 3.1.1

**Comment by david-batranu** [comment]
This is caused by the https://github.com/plone/Products.CMFEditions/commit/a8c0b41c5ca6fcc0e7d76d0b5a5cf4b9512627fb commit which added Python 3 compatibility.

**Comment by gp54321** [comment]
You can revert this regression that will probably not be solved in 5.1.2 with the ZMI:
portal_skins/CMFEditions/revertversion/manage_main
then customize, remove 'import six' and replace 'six.text_type' by 'unicode' and save.

**Comment by mauritsvanrees** [comment]
This was fixed in https://github.com/plone/Products.CMFEditions/pull/59 and is in release 3.2.2 from September 2018, which is included in Plone 5.1.4 and higher.
